internal/applicationsnapshot/report.go

```diff
@@ -67,6 +69,7 @@ type hacbsReport struct {
 	Failures  int       `json:"failures"`
 	Warnings  int       `json:"warnings"`
 	Result    string    `json:"result"`
+	Note      string    `json:"note"`
```

Comment:
We have to be careful with the structure of the hacbs report because it's something that is agreed on by different teams. I'd be hesitant to add a new field without consulting them. `Note` is something that is in the spec, which is why I used it. 

For now, I added a commit to remove the public key from the hacbs result. We still have the information in the logs via the yaml output. Let's wait until there's an explicit need for it.